### ISSUE TITLE
Accessibility: Domain request form State or territory Election office screen reader output

### ISSUE THREAD

**Issue by AnnaGingle** [cisagov/manage.get.gov#2570]
### Problem
On the domain request form for States or territories, the election office page has a `Is your organization an election office? ` radio field.

The ANDI / screen reader output is `Is election board * Yes not checked, required, 1 of 2`. The `Is election board *` portion should reference the field label (`Is your organization an election office?`). This fails section 508 success criteria 1.3.1-programmatic-label.

![Image](https://github.com/user-attachments/assets/d9b0533b-1b1a-4504-b26d-42eb638690c0)

Solution
The screen reader / ANDI output should be:
`Is your organization an election office? * Yes not checked, required, 1 of 2`

**Comment by AnnaGingle** [comment]
This is a dev ticket

**Comment by AnnaGingle** [comment]
This issue will be completed as a part of issue #2848
